srtcore/buffer_rcv.h

```diff
@@ -161,11 +161,11 @@ class CRcvBuffer
     /// Parameters (of the 1st packet queue, ready to play or not):
     /// @param [out] tsbpdtime localtime-based (uSec) packet time stamp including buffering delay of 1st packet or 0 if
     /// none
-    /// @param [out] passack   true if 1st ready packet is not yet acknowleged (allowed to be delivered to the app)
-    /// @param [out] skipseqno -1 or seq number of 1st unacknowledged pkt ready to play preceeded by missing packets.
+    /// @param [out] passack   true if 1st ready packet is not yet acknowledged (allowed to be delivered to the app)
+    /// @param [out] skipseqno -1 or seq number of 1st unacknowledged pkt ready to play proceeded by missing packets.
```

Comment:
Idem, "preceded".

Reply:
This is more for @stevomatthews to judge. The intent was passive: "a packet preceded by missing packets".

Reply:
```suggestion
    /// @param [out] skipseqno -1 or sequence number of 1st unacknowledged packet (after one or more missing packets) that is ready to play.
```